.travis.yml

```diff
@@ -17,6 +16,9 @@ before_install:
   - "export DISPLAY=:99.0"
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
+install:
```

Comment:
i see.. thanks. Maybe i will add that in there then. I use npm 3.x locally these days so i haven't seen those problems.